airflow/www/templates/airflow/ti_log.html

```diff
@@ -36,5 +41,44 @@ <h4>{{ title }}</h4>
         <pre id="attempt-{{ loop.index }}">{{ log }}</pre>
       </div>
     {% endfor %}
+    <img id="loading" alt="spinner" src="{{ url_for('static', filename='loading.gif') }}"
+         style="margin: auto; display: none;">
   </div>
+  <button class="btn btn-default pull-left refresh_button" style="margin-bottom: 10px;">
+      <span class="glyphicon glyphicon-refresh" aria-hidden="true" title="Refresh Logs"></span>
+  </button>
 {% endblock %}
+
+{% block tail %}
+{{ super() }}
+<script>
```

Comment:
So this shows errors in the notification bar at the top?

Reply:
yes